src/raftstore/store/peer.rs

```diff
@@ -1000,77 +996,71 @@ impl Peer {
         cmd_resp::bind_term(&mut resp, self.term());
         cb.call_box((resp,));
 
-        Ok(exec_result)
+        exec_result
     }
 
     pub fn term(&self) -> u64 {
         self.raft_group.raft.term
     }
 
+    // apply operation can fail as following situation:
```

Comment:
Yeah. It still helps to list some error for clarification.
If it's hard to clarity, it's hard to understand. If it's hard to understand, bug is likely to happen. What is an error that "occur on all store", a new guy who is assigned a task to add some steps into this function would wonder. And this scenario probably leads to bugs.